apps/supernova/src/lib/createAlertsSlice.tsx

```diff
@@ -5,20 +5,67 @@
 
 import { produce } from "immer"
 import { countAlerts } from "./utils"
+import { Filter } from "./createFiltersSlice"
+
+export interface AlertsSlice {
+  alerts: AlertsState
+}
+
+interface AlertsState {
+  items: AlertItem[]
+  itemsFiltered: AlertItem[]
+  totalCounts: AlertCounts
+  severityCountsPerRegion: Record<string, SeverityCounts>
+  regions: string[]
+  regionsFiltered: string[]
+  enrichedLabels: string[]
+  updatedAt: number | null
+  actions: AlertsActions
+}
+
+export interface AlertItem {
+  fingerprint: string
+  labels: Record<string, string>
+  [key: string]: any
+}
+
+export interface AlertCounts {
+  total: number
+  critical: number
+  [key: string]: number
+}
+
+export interface SeverityCounts {
+  total: number
+  critical?: { total: number; suppressed: number }
+  warning?: { total: number; suppressed: number }
+  [key: string]: any
+}
+
+export interface AlertsActions {
+  setAlertsData: (data: {
+    items: AlertItem[]
+    counts: { global: AlertCounts; regions: Record<string, SeverityCounts> }
+  }) => void
+  filterItems: () => void
+  setFilteredItems: (items: AlertItem[]) => void
+  setRegionsFiltered: (regions: string[]) => void
+  updateFilteredCounts: () => void
+  getAlertByFingerprint: (fingerprint: string) => AlertItem | undefined
+}
 
 const initialAlertsState = {
   items: [],
   itemsFiltered: [],
-  totalCounts: {}, // { total: number, critical: number, ...},
+  totalCounts: { total: 0, critical: 0 }, // { total: number, critical: number, ...},
```

Comment:
Why do you do a partial initialization here? The way it reads now it suggests that only the `total` and `critical` keys exist, but that's not true. The other severities are also counted (`warning` and `info`). So if you add initialization here it should be for all otherwise it might be confusing